tests/c.test

```diff
@@ -103,6 +103,7 @@
 00154  common/sp_after_cast.cfg                   c/casts.c
 00155  c/ben_078.cfg                              c/cast_brace.c
 
+00200  c/nl_collapse_empty_body_ignore_functions.cfg  c/empty_functions.c
```

Comment:
00200 is not very lucky, as we try to keep the numbers in order.
156 untill 159 are free, or 200 at another place in the file

Reply:
Done